### ISSUE TITLE
on_conflict_do_update and on_conflict_do_nothing returns None it typing, but actually returns 'Insert' 

### ISSUE THREAD

**Issue by BlackLIonz** [sqlalchemy/sqlalchemy2-stubs#212]
**Describe the bug**
Using on_conflict_do_update method with  returning and PyCharm  highlight return with "Cannot find reference 'returning' in 'None'" 

**Expected behavior**
Don't see any highlight.

**To Reproduce**
Install packages, and try to use 
``` python
insert(self.table).on_conflict_do_update(
      index_elements=(self.table.c.id,),
      set_={
          'count': self.table.c.count + insert_query.excluded.count,
      },
  ).returning(self.table.c.id)
```

**Error**

```
"Cannot find reference 'returning' in 'None'"
```

**Proposition**
Change:
``` python
def on_conflict_do_update(
        self,
        constraint: Optional[Any] = ...,
        index_elements: Optional[Any] = ...,
        index_where: Optional[Any] = ...,
        set_: Optional[Any] = ...,
        where: Optional[Any] = ...,
    ) -> 'Insert': ...
    def on_conflict_do_nothing(
        self,
        constraint: Optional[Any] = ...,
        index_elements: Optional[Any] = ...,
        index_where: Optional[Any] = ...,
    ) -> 'Insert': ...
```

**Versions.**
 - OS: Ubuntu 20.4
 - IDE: PyCharm Professional 2022.1
 - Python: 3.8
 - SQLAlchemy: 1.4.32
 - mypy: 0.782
 - SQLAlchemy2-stubs: 0.0.2a21

**Have a nice day!**

**Comment by CaselIT** [comment]
Hi, thanks for reporting.
Current effort is focused on typing the library for v2, but if can provide a pr we can merge and release

**Comment by BlackLIonz** [comment]
Can you give me contribution access? I cannot push and make PR

**Comment by CaselIT** [comment]
The usual way if to fork a repository, make changes there then create a PR

**Comment by CaselIT** [comment]
released as v0.0.2a22
